docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,23 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.2.1
+2021-03-04
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Upgrades
+
+- [Docker Engine 20.10.5](https://docs.docker.com/engine/release-notes/#20105)
+
+### Bug fixes and minor changes
+
+- Fixed a problem starting Docker Desktop 3.2.0 during a canary release. Fixes [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406)
```

Comment:
I don't think we've talked about canary releases publicly. Suggest "... 3.2.0 after an update"

Reply:
Fixed an issue that sometimes caused Docker Desktop to fail to start after upgrading to version 3.2.0. Fixes [docker/for-mac#5406](https://github.com/docker/for-mac/issues/5406).

Reply:
@usha-mandya Do you prefer "updating" or "upgrading"? I thought we tried to reserve "upgrade" for getting a paid plan, but I'm not sure we're very careful about it.

Reply:
That's a good point. We've used both 'update' and 'upgrade' previously. Agree that we should use 'update' in this context. This is also consistent with the UI (Check for Updates). Thanks for flagging.

Reply:
I've 'updated' my previous comment ;)